docs/pages/en-us/Client-API.md

```diff
@@ -179,10 +183,10 @@ The `AuditEventBuilder` object is created using the `createEventBuilder()` metho
 	// Add Event Parameters
 		// Add an Event Parameter for holding the Document ID that was deleted
 	auditEventBuilder.addEventParameter("docId", null, docId);
-		// Add an Event Parameter for holding the User who authorised the deletion of the document. Add length constraints for this parameter as it is of type String
-	auditEventBuilder.addEventParameter("authorisedBy", null, authorisedBy, 1, 256);
+		// Add an Event Parameter for holding the User who authorised the deletion of the document.
```

Comment:
@CJComac I would def re-introduce these as they appear to be still needed. They are also used in the Getting Started guide anyhow.